l3_footstep_planning_plugins/src/std/world_model/traversability_map_model.cpp

```diff
@@ -94,7 +98,7 @@ bool TraversabilityMapModel::isAccessible(const Foothold& foothold) const
 
   FootInfo foot_info;
   RobotModel::description()->getFootInfo(foothold.idx, foot_info);
-  Vector2 foot_size = Vector2(foot_info.size.x(), foot_info.size.y());
+  Vector2 foot_size = Vector2(foot_info.size.x() + foothold_margin_x_, foot_info.size.y() + foothold_margin_y_);
```

Comment:
Please store `foot_size` as private member like `upper_body_size_ ` to save computations on each callback,